### DISCUSSION TITLE
TailTidy - a library that has as intention to help you organize your Tailwind classes

### DISCUSSION THREAD

**Discussion by danvc** [tailwindlabs/tailwindcss#10724] [Help]
Hi guys!

I'm implementing a library in order to help organize classNames when we're dealing with a component that has dozens of classes that should be applied according the breakpoint.

An example on how to "organize" TailwindCSS classes:
````
  const classes = tidyUp({
    // component Title
    title: {
      // well, whant the breakpoint is small, I want to show a red background with the font-size small
      sm: {
        text: "text-sm",
        bg: "bg-red",
      },
      // but when the breakpoint reach the md size, I want to display the background with the color orange... the font-size will be normal
      md: {
        text: "md:text-base",
        bg: "md:bg-orange",
      },
      // but when the breakpoint reach lg, ah man, then, we need to rock this world and change the background color to blue and the font large
      lg: {
        text: "lg:text-lg",
        bg: "lg:bg-blue",
      },
    },
    // component Pragraph
    paragraph: {},
  });

````
Of course this example is stupid, however, the library is not stupid, mainly when you're dealing with elements that has dozens of classes. Urgh!

Use

````
<div className={classes.title.getClasses()}>Test</div>
````

If you work with huge components, you know how hard is to organize the classes in order to get it properly structured. Mainly when it should support different breakpoints
That was the reason why I started this project:
https://github.com/danvc/tailtidy

**My current problem** is that, even defining the classes properly, it IS setting the right classNames, however, the classes are not being injected, I mean, it's not getting effect.

Any thoughts on how to make it work?

**Comment by wongjn** [comment]
Consider checking the file(s) that contain instances of your example code are covered by the file globs in the Tailwind `content` configuration.
